src/syslog/syslog.c

```diff
@@ -64,3 +63,22 @@ void syslog_transport_close(syslog_transport_t *transport) {
     }
     transport->is_open = false;
 }
+void dns_found_cb(const char *name, const ip_addr_t *ipaddr, void *callback_arg) {
```

Comment:
/nitpick/ Empty line between functions please 😁

Reply:
And I don't see the function being used anywhere. I guess you've forgotten to remove it :)